frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -37,12 +38,67 @@ import { SurveysDisabledBanner, SurveySettings } from './SurveySettings'
 import { getSurveyStatus, surveysLogic, SurveysTabs } from './surveysLogic'
 
 export const scene: SceneExport = {
-    component: Surveys,
+    component: SurveysWithMaxTool,
     logic: surveysLogic,
     settingSectionId: 'environment-surveys',
 }
 
-export function Surveys(): JSX.Element {
+function SurveysWithMaxTool(): JSX.Element {
+    const {
+        data: { surveys },
+    } = useValues(surveysLogic)
+
+    const { loadSurveys } = useActions(surveysLogic)
+    const { user } = useValues(userLogic)
+
+    return (
+        <MaxTool
+            name="create_survey"
+            displayName="AI Survey Creator"
+            initialMaxPrompt="Create a survey to "
+            suggestions={[
+                'Create an NPS survey for customers who completed checkout',
+                'Create a feedback survey asking about our new dashboard',
+                'Create a product-market fit survey for trial users',
+                'Create a quick satisfaction survey for support interactions',
+            ]}
+            context={{
+                existing_surveys: surveys
+                    .filter((survey) => getSurveyStatus(survey) === ProgressStatus.Running)
+                    .map((survey) => ({
+                        id: survey.id,
+                        name: survey.name,
+                        type: survey.type,
+                        questions_count: survey.questions?.length || 0,
+                        status: getSurveyStatus(survey),
+                    })),
+                total_surveys_count: surveys.length,
+                user_id: user?.uuid, // Pass the actual user ID to the backend
```

Comment:
if the user isnt loaded, will the API error? should we prevent the submission instead?